packages/core/src/StateNode.ts

```diff
@@ -305,7 +305,7 @@ export class StateNode<
         const resolvedSrc =
           typeof src === 'string'
             ? src
-            : `xstate.source.${createInvokeId(this.id, i)}`;
+            : `xstate.invoke.${createInvokeId(this.id, i)}`;
```

Comment:
We might want to do a similar thing to `spawnChild` action in the future so I think it makes sense to use `invoke` here and not a generic `source`. This might become important for `resolveReferencedActor`